js/activity.js

```diff
@@ -5924,6 +5927,16 @@ class Activity {
             
             this._create2Ddrag();
 
+            setTimeout(() => {
+                console.log("dom is ready");
+                const urlParams = new URLSearchParams(window.location.search);
```

Comment:
Remind me what this does?

Reply:
this code checks if the url contains any key with `roomID`. if yes, it extracts its value and, initiate the collaboration and create the room with same `roomID`.